src/idl_parser.cpp

```diff
@@ -1635,3 +1635,78 @@ Offset<reflection::Type> Type::Serialize(FlatBufferBuilder *builder) const {
 }
 
 }  // namespace flatbuffers
+
+
+extern "C" {
+  FLATBUFFERS_EXPORT flatbuffers::Parser* flatbuffers_parser_new()
+  {
```

Comment:
thanks pointing this, I'll re-read google c++ style guide.
maybe still contains not compliant codes. i'll check with cpplint.py